tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -0,0 +1,153 @@
+import os
+import copy
+
+
+import etcd
+import time
+import socket
```

Comment:
Use gevent.socket instead of just socket

http://www.gevent.org/gevent.socket.html

It is a drop in replacement of socket. Same functions to send and receive